sdk/ml/azure-ai-ml/azure/ai/ml/entities/_builders/base_node.py

```diff
@@ -204,6 +204,40 @@ def name(self, value):
             value = value.lower()
         self._name = value
 
+    def get_predecessors(self) -> List["BaseNode"]:
```

Comment:
@wangchao1230 Do you think we should add an public API like this?

Reply:
+1 for this, not very sure whether there will be other usage than Det.Ch.

Reply:
Offline synced, removed the public API